Docs/source/acknowledgements.rst

```diff
@@ -0,0 +1,6 @@
+Acknowledgements
```

Comment:
Maybe even call this "Funding"?

Reply:
I feel like Acknowledgements encompasses both, funding information is found in the acknowledgement section of a paper, so I think "acknowledgement" is general enough, isn't it?

Reply:
Renamed it "Funding and Acknowledgements"